invertedai_cpp/invertedai/api.h

```diff
@@ -12,12 +12,31 @@
 
 namespace invertedai {
 
+/**
+ * Wrap the REST API "location_info".
+ * Provides static information about a given location.
+ */
```

Comment:
Some information about the parameters of the function is required.
e.g.
/**
@param location_info_request ....
@param session ...
@return What is returned....
@see initialize
@see drive
*/